<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1252</width>
    <height>888</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>460</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>180</red>
        <green>180</green>
        <blue>180</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>180</red>
        <green>180</green>
        <blue>180</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>180</red>
        <green>180</green>
        <blue>180</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>180</red>
        <green>180</green>
        <blue>180</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Fira Code</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Polyline Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="redo">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>310</y>
      <width>141</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>redo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="undo">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>260</y>
      <width>141</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>undo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="import_gpx">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>210</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>import gpx</string>
    </property>
   </widget>
   <widget class="QPushButton" name="import_polyline">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>160</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>import polyline</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="routes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>701</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>name</string>
     </property>
     <property name="font">
      <font>
       <family>Fira Code</family>
       <pointsize>9</pointsize>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="info">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>511</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>parameter</string>
     </property>
     <property name="font">
      <font>
       <family>Fira Code</family>
       <pointsize>9</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>value</string>
     </property>
     <property name="font">
      <font>
       <family>Fira Code</family>
       <pointsize>9</pointsize>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="delete_route">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>360</y>
      <width>141</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>delete route</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_point">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>410</y>
      <width>141</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>delete point</string>
    </property>
   </widget>
   <widget class="QChartView" name="widgetA" native="true">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>300</y>
      <width>511</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="count_turns">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>110</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>count turns</string>
    </property>
   </widget>
   <widget class="QPushButton" name="count_slopes">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>60</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>count slopes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="count_desc_asc">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>10</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>count desc asc</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="points">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>371</width>
      <height>691</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>latitude</string>
     </property>
     <property name="font">
      <font>
       <family>Fira Code</family>
       <pointsize>9</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>longitude</string>
     </property>
     <property name="font">
      <font>
       <family>Fira Code</family>
       <pointsize>9</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>elevation</string>
     </property>
     <property name="font">
      <font>
       <family>Fira Code</family>
       <pointsize>9</pointsize>
      </font>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <family>Courier New</family>
    </font>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>PyQt5.QtChart</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
